examples/spines/spines.py

```diff
@@ -8,6 +8,10 @@
 - normal Axes, with spines on all four sides;
 - an Axes with spines only on the left and bottom;
 - an Axes using custom bounds to limit the extent of the spine.
+
+Each `.axes.Axes` has a list of `~.Spine` objects, accessible
```

Comment:
```suggestion
Each `.axes.Axes` has a list of `.Spine` objects, accessible
```
The only purpose of `~` is shortening link text to the last component of the given qualified name. So `~.Something` is exactly the same as `.Something` and the latter is easier to read in the raw docstring.

Reply:
I mildly disagree - `~.Spine` makes it clear to me that its relative to "home", eg the top level of the library, whereas `.Spine` makes me think its a submodule of the current module.  I appreciate that's not how sphinx handles it, but when reading the raw docstring that's how I always think about it.

Reply:
Unfortunately, associating "~" with "home" is a false analogy. Coneceptually, "~" has nothing to do with the "path"/fully qualified name. It only affects the displayed text of the link. I advise against trying to use it for other purposes.

Reply:
Duh, sorry, you are right.  I should have my coffee before reading GitHub.